src/lib.rs

```diff
@@ -71,8 +69,8 @@ impl AttestationDocument {
                 )
             })?;
 
-        let verifier = rustls::AllowAnyAuthenticatedClient::new(root_store);
-        let _verified = verifier.verify_client_cert(&certs).map_err(|err| {
+        let verifier = rustls::server::AllowAnyAuthenticatedClient::new(root_store);
+        let _verified = verifier.verify_client_cert(&rustls::Certificate(document.certificate.clone()), &certs[..], std::time::SystemTime::now()).map_err(|err| {
```

Comment:
```suggestion
        let _verified = verifier.verify_client_cert(&rustls::Certificate(document.certificate.clone()), &certs, std::time::SystemTime::now()).map_err(|err| {
```

Is `[..]` really needed here?